scripts/qa_crowbarsetup.sh

```diff
@@ -3570,11 +3571,11 @@ function oncontroller_manila_generic_driver_setup()
 
 function oncontroller_magnum_service_setup ()
 {
-    # (mmnelemane): Replace this Fedora image with a suitable SLES image when available
-    local service_image_name=magnum-service-image.qcow2
-    local service_image_url=http://clouddata.cloud.suse.de/images/$(uname -m)/other/$service_image_name
+    local arch=$(uname -m)
+    local service_image_name=SLE12SP1-JeOS-k8s-magnum.qcow2
```

Comment:
I personally would prefer to have configuration in code rather than on the server :)